app/src/main/java/com/example/grutor/Fragments/LessonsFragment.java

```diff
@@ -85,6 +90,54 @@ public void onClick(View v) {
                 }
             }
         });
+        try {
+            queryStudentLessons();
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        lessonsAdapter = new LessonAdapter(getContext(), studentLessons);
+        rvLessons.setAdapter(lessonsAdapter);
+        tablLesssons.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
+            @Override
+            public void onTabSelected(TabLayout.Tab tab) {
+                if (tab.getText().toString().equals("Student")) {
```

Comment:
Use tab id instead of compare the text. if the app support multi-language, it doesn't work.